source/release-notes.txt

```diff
@@ -10,6 +10,31 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.44.5
+----------------
+
+*Released October 14, 2024*
+
+New Features:
+
+- Add polling for regular indexes (:issue:`COMPASS-8214`)
+- Allow editing name, color, and favorite checkbox of connected 
+  connections (:issue:`COMPASS-8160`)
+- Display banner on the connection form when 
```

Comment:
[suggestion] [style guide](https://www.mongodb.com/docs/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/#use-a-consistent-sentence-structure) - start the sentence with the conditional clause, so users can see if the condition applies to them first.

```suggestion
- When Protect Connection String Secrets is enabled, display banner on the connection form 
```